src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,6 +1,28 @@
 import React from 'react';
 import './PostInfo.scss';
+import { UnitedPost } from '../../react-app-env';
+import { CommentList } from '../CommentList/CommentList';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+type Props = {
+  currentPost: UnitedPost;
+};
+
+export const PostInfo: React.FC<Props> = ({ currentPost }) => (
+  <div className="PostInfo" data-cy="post-info">
+    <h3 className="PostInfo" data-cy="post-title">{currentPost.title}</h3>
+    <p data-cy="post-body">{currentPost.body}</p>
+    <h4>User information:</h4>
+    <ul>
+      <li data-cy="user-name">
```

Comment:
You lost  `UserInfo` component)

The `PostInfo` accepts a `post` object (with `user` and `comments` properties added before) and renders a `title`, a `body`, a `UserInfo` and a `CommentList`.

The `UserInfo` accepts a `user` object and renders at least a `name` and an `email`.